src/main/webapp/app/exercises/modeling/participate/modeling-submission.service.ts

```diff
@@ -52,7 +52,7 @@ export class ModelingSubmissionService {
      * @param {any?} req - Request option
      * @param correctionRound correctionRound for which to get the Submissions
      */
-    getModelingSubmissionsForExerciseByCorrectionRound(exerciseId: number, req?: any, correctionRound = 0): Observable<HttpResponse<ModelingSubmission[]>> {
+    getSubmissions(exerciseId: number, req?: any, correctionRound = 0): Observable<HttpResponse<ModelingSubmission[]>> {
```

Comment:
Q: What's the rationale behind the re-naming?

Reply:
Having too long names leads to mental overhead on order to understand their core functionality.

If I get a submission using a ProgrammingSubmissionService, I know that it is a programming Submission. Looking at the parameters I know that it is by exerciseId and correctionRound.

So ProgrammingSubmissionService.getProgrammingSubmissionByExerciseIdByCorrectionRound becomes ProgrammingSubmissionService.getSubmission